Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -2155,7 +2156,14 @@ def perform_long_running_loop(client: Client, offenses_per_fetch: int, fetch_mod
                               asset_enrich: bool, incident_type: Optional[str], mirror_direction: Optional[str],
                               first_fetch: str, mirror_options: str):
     is_reset_triggered()
-    context_data, _ = get_integration_context_with_version()
+    context_data, version = get_integration_context_with_version()
+    last_highest_id_last_run = demisto.getLastRun().get(LAST_FETCH_KEY, 0)
```

Comment:
load integration context only at the beginning of the long running . otherwise keep the integration context and the last run in the memory. pass it between iterations